crates/ekore/Cargo.toml

```diff
@@ -9,3 +9,4 @@ license = "GPL-3.0-or-later"
 
 [dependencies]
 num = "0.4.1"
+bibliothek = "0.1.0"
```

Comment:
I would make this a dev-dep, do you believe it would be a problem?
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
Nono, I also prefer the
```
:cite:`bla2023`
```
option (even though it looks more like ReST than MD).

My point was mainly about making it a development dependency, because you do not need to compile the actual library, but only for the docs.
I'm not sure if it would fully work (e.g. on crates.io), but it should be sensible, so it might. I'd just try.

Reply:
> all in all I have to say, I'm disappointed by Rust - not having a built-in citation system is a no go for academics; we want to and have to honour other people ...

Fair enough :)

Reply:
> even though it looks more like ReST than MD

it *is* the copied rst and thus not working :upside_down_face: the actual proposal is
```rust
/// Implements Eq. (3.4) of [Moch:2004pa][cite Moch:2004pa].
///
/// [cite Moch:2004pa]: ./../../../../bib/index.html#moch2004pa
```
plus the `bib.rs`

Reply:
But I don't like all this non-sense:
![grafik](https://github.com/NNPDF/eko/assets/6421325/362d12f4-2e04-43d0-bd8c-17192a42f07c)
the ratio relevant information/non-sense is 2/27

Reply:
Then we could try patching `bibliothek` to use functions rather than structs. A function has no implementation-like add-on, so it should not be cluttered, e.g. [`gamma_gg`](file:///media/alessandro/moneybin/Projects/NNPDF/eko/crates/target/doc/ekore/anomalous_dimensions/unpolarized/spacelike/as1/fn.gamma_gg.html)

Reply:
I asked the author for a new tag ...